### PR TITLE
Feature/merge closures instead of grouping in array

### LINKED ISSUE #4: parsing issue occurs in case build.gradle file contains several 'android' closures
Hello! The issue occurs when I've tried to execute `parseFile` method on [this build.gradle file](https://gist.github.com/mateusreis/15c0370766536634ac34003eafb80bef)
Parsing result represents `android` prop as array instead of object.

### PR DESCRIPTION
**Pull request by karllindmark**
Closes #4 reported by @sergey-akhalkov

@geof90, any thoughts on the proposed solution? The tests are passing, but I'd like your two cents too. 👍

### CODE REVIEW

**Comment by karllindmark** [comment]
I thought `Object.assign()` was ES5, but apparently not, so I swapped it out for `deep-assign` by @sindresorhus. Tests are passing on Travis again. :-)

File: lib/parser.js [hunk 1]
```diff
@@ -123,7 +132,11 @@ function deepParse(chunk, state, keepFunctionCalls) {
       if (SPECIAL_KEYS.hasOwnProperty(currentKey)) {
         out[currentKey] = SPECIAL_KEYS[currentKey](chunk, state);
       } else {
-        out[currentKey] = deepParse(chunk, state, keepFunctionCalls);
+        if (out[currentKey]) {
+          out[currentKey] = deepAssign({}, out[currentKey], deepParse(chunk, state, keepFunctionCalls, skipEmptyValues));
+        } else {
+          out[currentKey] = deepParse(chunk, state, keepFunctionCalls, skipEmptyValues);
+        }
```
Comment (geof90) [inline lib/parser.js]: nitpick: we could do `else if` and `else` and avoid one level of nesting here.

Comment (karllindmark) [inline lib/parser.js]: Fixed in 9f192b6!

**Comment by karllindmark** [comment]
I'd like to get some feedback from @sergey-akhalkov before I merge this, to see if it works for him too. :-)

**Pull request merged**